5_analyze_data/notebooks/Heatmap_analysis/nf1_complexheatmap.r

```diff
@@ -0,0 +1,155 @@
+suppressPackageStartupMessages(library(ComplexHeatmap))
+suppressPackageStartupMessages(library(dplyr))
+
+# Set paths and constants
+input_data_dir <- file.path("..", "..", "..", "4_processing_features", "data")
+output_figure_dir <- "figures"
+
+cp_heatmap_file <- file.path(output_figure_dir, "cp_complex_heatmap.pdf")
+dp_heatmap_file <- file.path(output_figure_dir, "dp_complex_heatmap.pdf")
+
+# Set heatmap colors
+well_cols = c(
+    "C6" = "#E1DAAE",
+    "C7" = "#FF934F",
+    "D6" = "#CC2D35",
+    "D7" = "#058ED9",
+    "E6" = "#848FA2",
+    "E7" = "#2D3142",
+    "F6" = "#FFC857",
+    "F7" = "#5f7a12"
+)
+genotype_cols = c(
+    "Null" = "#785EF0",
+    "WT" = "#DC267F"
+)
+
```

Comment:
How did you decide on these color specifically? Would it not be easy to use a cmap or preset color palette to use?

Reply:
This could also be an R related thing I don't know about now that I think of it.

Reply:
I decided on these colors for a separate project :joy: (https://github.com/broadinstitute/profiling-resistance-mechanisms/blob/master/3.resistance-signature/scripts/nbconverted/7.profile-heatmaps.r)

I picked them because they are compatible with different kinds of color blindness. I don't remember exactly, but I think I used https://colorbrewer2.org